README.md

```diff
@@ -48,24 +71,82 @@ Then update to your own private key.
 
 ## Print wallet balances
 
-This script will print your wallet balances for each chain.
+To print your wallet balances for each chain, run:
 
 ```bash
 npm run check-balance [local|testnet]
 ```
 
-If not specified, this will print balances of the wallet for testnet.
+The aforementioned command pertains to specifying the intended environment for a project to execute on. It provides the option to choose between local and testnet environments by appending either `local` or `testnet` after the command. 
+
+An example of its usage is demonstrated as follows: `npm run check-balance local` or `npm run check-balance testnet`. It is noteworthy that the script defaults to testnet if no environment is explicitly specified.
 
 ## Running the local chains
 
+To start the local chains, run the following command:
+
```

Comment:
This should go before the "print wallet balance" section.

Reply:
Updated